packages/core/src/modules/mdoc/MdocOptions.ts

```diff
@@ -0,0 +1,56 @@
+import type { Mdoc } from './Mdoc'
+import type { Key } from '../../crypto/Key'
+import type { DifPresentationExchangeDefinitionV2 } from '../dif-presentation-exchange'
+import type { ValidityInfo, MdocNameSpaces } from '@protokoll/mdoc-client'
+
+export type { MdocNameSpaces } from '@protokoll/mdoc-client'
+
+export type MdocVerifyOptions = {
+  trustedCertificates?: [string, ...string[]]
+  now?: Date
+}
+
+export type MdocOpenId4VpSessionTranscriptOptions = {
+  responseUri: string
+  clientId: string
+  verifierGeneratedNonce: string
+  mdocGeneratedNonce: string
+}
+
+export type MdocDeviceResponseOpenId4VpOptions = {
+  mdocs: [Mdoc, ...Mdoc[]]
+  presentationDefinition: DifPresentationExchangeDefinitionV2
+  deviceNameSpaces?: MdocNameSpaces
+  sessionTranscriptOptions: MdocOpenId4VpSessionTranscriptOptions
+}
+
+export type MdocDeviceResponseVerifyOptions = {
+  mdoc: Mdoc
+  trustedCertificates?: [string, ...string[]]
+  sessionTranscriptOptions: MdocOpenId4VpSessionTranscriptOptions
+  /**
+   * The base64Url-encoded device response string.
+   */
+  deviceResponse: string
```

Comment:
Why do we need to add both the mdoc and the deviceResponse base64? The encoded device response also contains the mdoc right? 

If it's for internal APIs it's ok, but for public APIs I'd like to avoid options that could conflict (what if they don't match?)